catalog/transformer/catalog-transformer-preview/src/main/java/org/codice/ddf/transformer/preview/PreviewMetacardTransformer.java

```diff
@@ -36,10 +37,8 @@ public BinaryContent transform(Metacard metacard, Map<String, Serializable> argu
     if (metacard.getAttribute(Extracted.EXTRACTED_TEXT) != null
         && metacard.getAttribute(Extracted.EXTRACTED_TEXT).getValue() != null) {
       preview =
-          metacard
-              .getAttribute(Extracted.EXTRACTED_TEXT)
-              .getValue()
-              .toString()
+          StringEscapeUtils.escapeHtml4(
+                  metacard.getAttribute(Extracted.EXTRACTED_TEXT).getValue().toString())
               .replaceAll("[\n|\r]", "<br>");
```

Comment:
Now that we are escaping HTML, do we still need to strip the string `<br>`?

Reply:
Yeah, we should still do this. Any <br> in the text will still be displayed as <br> text because were escaping the html element, but we add in the <br> element for new lines after so that new lines are rendered appropriately